src/OpenRiaServices.Tools/Test/MsBuildHelper.cs

```diff
@@ -332,6 +334,8 @@ private class ErrorLogger : Microsoft.Build.Framework.ILogger
             public void Initialize(IEventSource eventSource)
             {
                 eventSource.ErrorRaised += (s, a) => this._errors.Add($"{a.File}({a.LineNumber},{a.ColumnNumber}): error {a.Code}: {a.Message}");
+                //eventSource.WarningRaised += (s, a) => this._errors.Add($"{a.File}({a.LineNumber},{a.ColumnNumber}): error {a.Code}: {a.Message}");
```

Comment:
Remove this?

Reply:
it can be removed, it was invaluable to troubleshooting msbuild related issues